docs/module/lag.md

```diff
@@ -82,7 +82,14 @@ The above diagram illustrates the three supported topologies:
 * 2:2 dual MLAG between 2 pairs of nodes (4 nodes in total)
 
 MLAG related parameters:
-* **lag.mlag.mac**: Used at node or interface level to configure the MAC address for the peer link. *Netlab* can auto-generate this, so it usually is not necessary to set this
+
+Global:
```

Comment:
Looking at the code, it seems to me that this has to be defined in the device features, not as a global attribute. Am I missing something?

Reply:
Globally as a device feature ;) Clarified

It implies the code does not support topologies in which some devices use ```loopback.ipv4``` and others use ```mgmt.ipv4``` - that could be generalized (by making it a node level attribute)